src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1844,20 +1844,68 @@ When HTTPS is set up, you can automatically redirect any HTTP request from
 users to the secured version of the protocol to make sure exchanged information
 is protected. It requires to enable `web.redirect_http_to_https`.
 
-Unlike other web applications, the Scheduler, the Resource Manager and the Studio web portals together with the
-ProActive Cloud Watch, Cloud Automation service, Job Planner and Notification service
-require manual configuration based on your HTTPS settings. The configuration
-files to edit are respectively in
+Finally, you will need to update the port number used in the property `jp.url`.
```

Comment:
I suggest to replace 'Finally' with 'Also'

Reply:
ok